velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -418,7 +452,9 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
         std::make_shared<RowVector>(pool_, type, nullptr, batchSize, children));
   }
 
-  return std::make_shared<core::ValuesNode>(nextPlanNodeId(), vectors);
+  return ProcessEmit(
```

Comment:
It doesn't make sense to apply emit to values node. Can we just make sure that values node is created with the right order of columns?

Reply:
I guess what you mean is if someone goes into the length of creating data in terms of a virtual table why add an emit later to just limit the columns manually added. Is this a correct interpretation?

Reply:
@vibhatha Indeed, I expect even most basic optimizers to optimize such plans to remove/reorder columns in the values node directly instead of keeping values node with more columns than needed and have an "emit" on top of it to remove some of the columns.

Reply:
Yes, that perfectly make sense. I will remove the emit function for such nodes.